code/modules/mob/mob_helpers.dm

```diff
@@ -256,19 +256,20 @@
 	for(var/datum/multilingual_say_piece/S in message_pieces)
 		S.message = stars(S.message, pr)
 
-/proc/slur(phrase, list/slurletters = ("'"))//use a different list as an input if you want to make robots slur with $#@%! characters
+/proc/slur(phrase, list/slurletters = ("'"), robot = FALSE) //use a different list as an input if you want to make robots slur with $#@%! characters
```

Comment:
```suggestion
/proc/slur(phrase, list/slurletters = ("'"), skip_replacement = FALSE) //use a different list as an input if you want to make robots slur with $#@%! characters
```
This should probably be a variable that can be used without being a robot

Reply:
I checked with Vi3, you have to replace if(!robot) with if(!skip_replacement) as well. Once you do that I'll definitely commit this

Reply:
I highly recommend not commiting from the website, because something might go wrong and it shows that you havent tested it. Best to change stuff in vscode then commit it yourself when someone makes a recommendation

Reply:
Testing worked. One sec.